pom.xml

```diff
@@ -182,7 +182,7 @@
         <air.test.jvm.additional-arguments>${air.test.jvm.additional-arguments.default}</air.test.jvm.additional-arguments>
 
         <!-- keep dependency properties sorted -->
-        <dep.airlift.version>316</dep.airlift.version>
+        <dep.airlift.version>317</dep.airlift.version>
```

Comment:
Why do we update airlift version in "Update parquet to 1.15.1" commit?

Reply:
my bad :) I'll add separate commit